analysis/druidferal/src/modules/racials/Shadowmeld.js

```diff
@@ -29,8 +30,8 @@ class Shadowmeld extends Analyzer {
     return {
       actual: this.correctUses / this.possibleUses,
       isLessThan: {
-        minor: 0.9,
-        average: 0.8,
+        minor: 0.7,
```

Comment:
Should probably leave these the same? Since it changes the color of the bar in the checklist right?

Reply:
True but it's way too strict towards the current importance of Shadowmeld, I'll change them to be different numbers but lower